courses/machine_learning/deepdive/09_sequence_keras/translate/translate_model.ipynb

```diff
@@ -0,0 +1,809 @@
+{
```

Comment:
<ul><li>pep 8</li><li>I don't find this cell very readable, but I don't  see right now how to improve on what you did here. Do you see a way to break/package the code in smaller chunks? Maybe separating the code that creates the individual layers from the code that composes these layers? The same kind of comments goes for the cell above. What do you think?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>